AspNetCore.Demo/AspNetCore.Demo.csproj

```diff
@@ -20,19 +20,15 @@
     <ProjectReference Include="..\AspNetCore\AspNetCore.csproj" />
     <ProjectReference Include="..\Common\Common.csproj" />
     <ProjectReference Include="..\FSharp\FSharp.csproj" />
+    <ProjectReference Include="..\IL\IL.csproj" />
   </ItemGroup>
 
   <Target Name="CopySdkReferenceAssemblies" AfterTargets="ResolveReferences">
     <ItemGroup>
-      <Content Include="@(Reference->'%(RootDir)%(Directory)%(Filename)%(Extension)')"
-               Link="@(Reference->'ref-assemblies\%(Filename)%(Extension)')"
-               Condition="$([System.String]::new('%(Filename)').StartsWith('System')) or '%(Filename)' == 'netstandard'">
+      <Content Include="@(Reference->'%(RootDir)%(Directory)%(Filename)%(Extension)')" Link="@(Reference->'ref-assemblies\%(Filename)%(Extension)')" Condition="$([System.String]::new('%(Filename)').StartsWith('System')) or '%(Filename)' == 'netstandard'">
```

Comment:
This formatting change (same below) looks unintentional -- can be reverted?

Reply:
Yep, sorry! Fixed.